server/views/pages/githubRepoRequestForm.njk

```diff
@@ -73,27 +73,28 @@
           text: "hmpps-template-typescript"
         }]
     }) }}
-      {{ govukRadios({
-      classes: "govuk-radios--inline",
-      idPrefix: "github_project_visibility",
+    {{ govukSelect({
+      id: "github_project_visibility",
       name: "github_project_visibility",
       value: submittedForm.github_project_visibility,
-      errorMessage: validationErrors | findError('github_project_visibility'),
-      fieldset: {
-        legend: {
-          text: "Github Repository Visibility",
-          classes: 'govuk-fieldset__legend--s'
-        }
+      label: {
+        text: "Github project visbility"
       },
       items: [
         {
-          text: "Internal",
-          value: "false"
+          value: "public",
+          text: "public"
```

Comment:
Might be worth capitalising the `text` just to make it as it was i.e. `Internal`, `Private` and `Public`

Reply:
Updated text as suggested